src/xtgeo/well/well1.py

```diff
@@ -29,9 +33,20 @@
 # METHODS as wrappers to class init + import
 
 
+def _data_reader_factory(file_format):
+    if file_format in ["rmswell", "irap_ascii"]:
+        return _well_io.import_rms_ascii
+    if file_format == "hdf":
+        return _well_io.import_hdf5_well
+    raise ValueError(
+        f"Unknown file format {file_format}, supported formats are "
+        "'rmswell', 'irap_ascii' and 'hdf'"
+    )
+
+
 def well_from_file(
     wfile: Union[str, Path],
-    fformat: Optional[str] = "rms_ascii",
+    fformat: Optional[str] = "irap_ascii",
```

Comment:
Why is the default changing?

Reply:
The default hasn't really changed as the code treats `rms_ascii` as `irap_ascii`. That being said, it is no need to change that as it might be a breaking change. Removed it.

Reply:
A note: `rms_ascii` is the only correct for wells from RMS. The old tool "irap" did not have this format. For maps and points, the formats from the old Irap tool is applied in RMS, hence "irap_ascii" and "rms_ascii" are there the same.

Reply:
> A note: rms_ascii is the only correct for wells from RMS. The old tool "irap" did not have this format. For maps and points, the formats from the old Irap tool is applied in RMS, hence "irap_ascii" and "rms_ascii" are there the same.

Should we change the default to `irap_ascii` then?

Reply:
This context might be good to have in a comment?